Funzione_Turbine_Darrieus_teoria_singolo_tubo_di_flusso

AngDasc opened the pull request:
Per lo sviluppo della stessa siamo in attesa delle lezioni teoriche. Nel frattempo chiediamo conferma per lo sviluppo del branch.

Commit:
Prova 2

EttoreSaetta commented:
> Per lo sviluppo della stessa siamo in attesa delle lezioni teoriche. Nel frattempo chiediamo conferma per lo sviluppo del branch.

@AngDasc Il branch è creato correttamente.

Commit:
- y assegnato per ottenere valori di a tramite l'integrale 10.12.
- Valori di a ottenuti.
- con "y" e "a" è stata trovata lambda.
 C'è un problema nel digramma cp vs lambda, esce una retta come risultato.

mgugliu committed:
Update Darreus_singolo.m

Problem solved.

mgugliu commented:
Problem solved. Tomorrow we are going to fix the formatting of the function.

Commit:
Function and main file works with coments. We are waiting for the next meeting.

Commit:
The function works, has coments. We are waiting for the nex meeting.

Commit:
Update 1

Commit:
Function update.  Input and output docs added.

Error in low values of lambda. 

Waiting for review.

AngDasc committed:
Add files via upload

Commit:
Function update. 
The function needs the alphamax controller.

Commit:
Merge commit 'b4a5c9338867da936fa9d35aae77b336107ecd9e'

AngDasc committed:
Add files via upload

EttoreSaetta commented on Turbine_single_tube.m:
```diff
@@ -0,0 +1,95 @@
+    
+%   Determination of tip speed ratio
+    lambda = y.*(1-a);
+    Lambda(i,1)=lambda;
```
You should move `Lambda(i,1)=lambda` inside the `if alpha < alphamax` statement.
With this correction you will solve the problem.

AngDasc replied:
The correction has been applied.

AngDasc committed:
Update Turbine_single_tube.m

Correction has been applied.

AngDasc committed:
Add files via upload

Function theory.

AngDasc committed:
Update Turbine_single_tube.m

AngDasc committed:
Update mainsingle.m

Main update to the last version with all the test case.

AngDasc committed:
Update mainsingle.m

AngDasc committed:
Rename mainsingle.m to main.m

AngDasc committed:
Delete main.m

AngDasc committed:
Delete Docs directory

AngDasc committed:
Add files via upload

Definitive main script.

AngDasc committed:
Add files via upload

Document in PDF

AngDasc committed:
Delete DeVries_data.mat

AngDasc committed:
Add files via upload

EttoreSaetta reviewed (approved):
LGTM
